xds/src/main/java/io/grpc/xds/XdsLoadReportClientImpl.java

```diff
@@ -342,6 +346,23 @@ private void cleanUp() {
     }
   }
 
+  /**
+   * Callbacks for passing information received from client load reporting responses to xDS load
+   * balancer, such as the load reporting interval requested by the traffic director.
+   *
+   * <p>Implementations are not required to be thread-safe as callbacks will be invoked in xDS load
+   * balancer's {@link SynchronizationContext}.
+   */
+  interface XdsLoadReportCallback {
```

Comment:
This interface should be inside `XdsLoadReportClient`.

Reply:
Done.